Formula/g/godap.rb

```diff
@@ -0,0 +1,24 @@
+class Godap < Formula
+  desc "Complete TUI (terminal user interface) for LDAP"
+  homepage "https://github.com/Macmod/godap"
+  url "https://github.com/Macmod/godap.git",
+    tag:      "v2.5.0",
+    revision: "a5201bc2426d2fbeb469b40c532d1974246d487e"
+  license "MIT"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}")
+    generate_completions_from_executable(bin/"godap",  "completion")
+  end
+
+  test do
+    assert_includes shell_output("#{bin}/godap -T 1 203.0.113.1 2>&1", 1),
+      "LDAP Result Code 200 \"Network Error\": dial tcp 203.0.113.1:389: i/o timeout"
```

Comment:
```suggestion
    output = shell_output("#{bin}/godap -T 1 203.0.113.1 2>&1", 1)
    assert_equal "LDAP Result Code 200 \"Network Error\": dial tcp 203.0.113.1:389: i/o timeout", output

    assert_match version.to_s, shell_output("#{bin}/godap version")
```

Reply:
The output includes a timestamp:
2024/05/14 09:46:30 LDAP Result Code 200 "Network Error": dial tcp 203.0.113.1:389: i/o timeout

How about:
```golang
-    assert_includes shell_output("#{bin}/godap -T 1 203.0.113.1 2>&1", 1),
-      "LDAP Result Code 200 \"Network Error\": dial tcp 203.0.113.1:389: i/o timeout"
+    output = shell_output("#{bin}/godap -T 1 203.0.113.1 2>&1", 1)
+    assert_includes output, "LDAP Result Code 200 \"Network Error\": dial tcp 203.0.113.1:389: i/o timeout"
+
+    assert_match version.to_s, shell_output("#{bin}/godap version")
```

Reply:
yeah, indeed, using `assert_match` should cover it.